api-docs/api-reference/methods/put-update-private-monitor-zone.rst

```diff
@@ -0,0 +1,75 @@
+|400                       |Bad request              |The system received an   |
+|                          |                         |invalid value in a       |
+|                          |                         |request.                 |
++--------------------------+-------------------------+-------------------------+
+|401                       |Unauthorized             |The system received a    |
+|                          |                         |request from a user that |
+|                          |                         |is not authenticated.    |
++--------------------------+-------------------------+-------------------------+
+|403                       |Forbidden                |The system received a    |
+|                          |                         |request that the user is |
+|                          |                         |not authorized to make.  |
++--------------------------+-------------------------+-------------------------+
+|404                       |Not Found                |The URL, entity, or      |
+|                          |                         |account requested is not |
+|                          |                         |found in the system.     |
++--------------------------+-------------------------+-------------------------+
+|500                       |Internal Server Error    |An unexpected condition  |
+|                          |                         |was encountered.         |
++--------------------------+-------------------------+-------------------------+
+|503                       |Service Unavailable      |The system is            |
+|                          |                         |experiencing heavy load  |
+|                          |                         |or another system        |
+|                          |                         |failure.                 |
++--------------------------+-------------------------+-------------------------+
+
+Request
+-------
+
+The following table shows the header parameters for the request:
+
++-----------------+----------------+-----------------------------------------------+
+|Name             |Type            |Description                                    |
++=================+================+===============================================+
+|X-Auth-Token     |String          |A valid authentication token with              |
+|                 |*(Required)*    |administrative access. For details, see        |
+|                 |                |:ref:`Get your credentials <get-credentials>`  |
++-----------------+----------------+-----------------------------------------------+
+
+.. note:: This operation does not accept a request body.
+
+**Example Update a private monitoring zone: JSON request**
+
+.. code::
+
+   {
+       <!--Todo: process Request example -->
+   }
+
+Response
+--------
+
+The following operation does not return a response body.
```

Comment:
Should it be "This operation" instead of "The following operation"?

Reply:
Corrected